src/lib.rs

```diff
@@ -323,7 +388,10 @@ mod tests {
     #[test]
     fn single_error_case() {
         custom_error!(MyError Bad="bad");
-        assert_eq!("bad", MyError::Bad.to_string())
+        assert_eq!("bad", MyError::Bad.to_string());
+
+        custom_error!(MyErrorStruct{} ="bad");
```

Comment:
Would it be possible to create new tests instead of re-using the existing ones ? Since the two cases test distinct parts of the codes, having them in separate tests would make debugging easier.